opm/material/fluidmatrixinteractions/EclHysteresisTwoPhaseLawParams.hpp

```diff
@@ -286,9 +289,9 @@ class EclHysteresisTwoPhaseLawParams : public EnsureFinalized
         Scalar SwKrnMdcImbibition = EffLawT::twoPhaseSatKrnInv(imbibitionParams(), krnMdcDrainage);
         deltaSwImbKrn_ = SwKrnMdcImbibition - krnSwMdc_;
 
-        Scalar pcMdcDrainage = EffLawT::twoPhaseSatPcnw(drainageParams(), pcSwMdc_);
-        Scalar SwPcMdcImbibition = EffLawT::twoPhaseSatPcnwInv(imbibitionParams(), pcMdcDrainage);
-        deltaSwImbPc_ = SwPcMdcImbibition - pcSwMdc_;
```

Comment:
are you sure that it is correct to remove this? this will effectively imply that the drainage curve is always used as the capillary pressure curve. (not sure if pc-hysteresis implemented yet, though. if not, a comment would be good.)